scripts/grafana-server/start-server

```diff
@@ -34,6 +34,17 @@ mkdir $PROV_DIR/dashboards
 cp ./scripts/grafana-server/custom.ini $RUNDIR/conf/custom.ini
 cp ./conf/defaults.ini $RUNDIR/conf/defaults.ini
 
+echo -e "Copying custom plugins from e2e tests"
+
+mkdir -p "$RUNDIR/data/plugins"
+# when running in a local computer
+if [ -d "./e2e/custom-plugins" ]; then
+  cp -r "./e2e/custom-plugins" "$RUNDIR/data/plugins"
+# when running in CI
+elif [ -d "../e2e/custom-plugins" ]; then
+  cp -r "../e2e/custom-plugins" "$RUNDIR/data/plugins"
+fi
+
```

Comment:
Should we remove these plugins on a "tear-down" event when E2E tests finish? 

Also, is this `start-server` script only used for setting up the server for tests?

Reply:
There's already a teardown that takes care of removing the whole `$RUNDIR`. Locally it is invoked whenever a new server is invoked. in CI doesn't matter because the instances get trashed after each run.